src/gui/sdlmain.cpp

```diff
@@ -2372,8 +2372,13 @@ bool GFX_StartUpdate(uint8_t * &pixels, int &pitch)
 	return false;
 }
 
-void GFX_EndUpdate(const uint16_t *changedLines)
+extern int64_t ticksDone;
+
+void GFX_EndUpdate(const uint16_t* changedLines)
 {
+	static int64_t cumulativeTimeSlept = 0;
```

Comment:
For new code, `lower_case_snake` local variables 

(There's still a mix of the some of the inconsistent prior upstream styles in here)

Reply:
By `slept`, you mean the CPU core was sleeping while the time was spent rendering?

Reply:
oops, copypasta :grin: